### DISCUSSION TITLE
Up to date guide for using AWS Amplify?

### DISCUSSION THREAD

**Discussion by JulioBarros** [vercel/next.js#11086] [Help]
Hi,
Does anyone know of an up to date guide to using AWS Amplify and NextJS? Is it even possible with GraphQL and (Cognito) Auth. I've gotten a variety of errors from css issues with aws-amplify-react to not being able to figure out how to have public and logged in pages. 

Any guides, tips, pointers are appreciated.

Thanks.

Julio

**Comment by leerob** [comment]
[This post](https://dev.to/rakannimer/server-side-rendered-real-time-web-app-with-next-js-aws-amplify-graphql-2j49) was updated Sep 2019 and [recently tweeted](https://twitter.com/AWSAmplify/status/1258440060858441729) by [@dabit3](https://twitter.com/dabit3) from AWS.
